eng/helix/content/mssql/InstallSqlServerLocalDB.ps1

```diff
@@ -31,7 +31,7 @@ $installerFilename = "SqlLocalDB.msi"
 $installerPath = "$intermedateDir\$installerFilename"
 Write-Host ""
 Write-Host "Downloading '$installerFilename' to '$installerPath'."
-& $PSScriptRoot\Download.ps1 'https://download.microsoft.com/download/9/0/7/907AD35F-9F9C-43A5-9789-52470555DB90/ENU/SqlLocalDB.msi' $installerPath
+& "$PSScriptRoot\..\..\..\scripts\Download.ps1" 'https://download.microsoft.com/download/9/0/7/907AD35F-9F9C-43A5-9789-52470555DB90/ENU/SqlLocalDB.msi' $installerPath
```

Comment:
I don't think that will work on Helix. The `content` folder is copied to the machine, and the `scripts` folder is copied to the machine. So the folder structure is likely different on Helix. One minute and I'll try to find what it should be.

Reply:
I think `$PSScriptRoot\..\Download.ps1` will work.

Reply:
Hmm, we seem to be doing this pretty extensively:
https://github.com/dotnet/aspnetcore/blob/c925f99cddac0df90ed0bc4a07ecda6b054a0b02/eng/helix/content/InstallNode.ps1#L37

https://github.com/dotnet/aspnetcore/blob/c925f99cddac0df90ed0bc4a07ecda6b054a0b02/eng/helix/content/InstallJdk.ps1#L48

We'll likely have to update elsewhere as well to avoid future issues.

Reply:
```suggestion
& "$PSScriptRoot\..\Download.ps1" 'https://download.microsoft.com/download/9/0/7/907AD35F-9F9C-43A5-9789-52470555DB90/ENU/SqlLocalDB.msi' $installerPath
```

Reply:
Thanks, will wait for the CI to pass. If it does, will update accordingly elsewhere.

Reply:
Curious was this just 100% broken, and mssql was working because it was on the machine or we just don't need it anywhere?

Reply:
>Hmm, we seem to be doing this pretty extensively:

Those two will be in the root directory where "Download.ps1" is located when on Helix so they're fine.

Reply:
> Curious was this just 100% broken, and mssql was working because it was on the machine or we just don't need it anywhere?

A few hours ago:
```
C:\h\w\B5AA09DF\w\BB490A08\e>call RunPowershell.cmd mssql\InstallSqlServerLocalDB.ps1   || exit /b 1 
"PS: Running 'C:\h\w\B5AA09DF\w\BB490A08\e\mssql\InstallSqlServerLocalDB.ps1' "
SQL Server 2016 Express LocalDB is already installed. Exiting without action. Call this script again with -Force to run the installation anyways.
```
Now:
```
C:\h\w\A5C508E5\w\A7D80973\e>call RunPowershell.cmd mssql\InstallSqlServerLocalDB.ps1   || exit /b 1 
"PS: Running 'C:\h\w\A5C508E5\w\A7D80973\e\mssql\InstallSqlServerLocalDB.ps1' "
Installing SQL Server 2016 Express LocalDB

Downloading 'SqlLocalDB.msi' to 'C:\h\w\A5C508E5\w\A7D80973\e\mssql\obj\SqlLocalDB.msi'.
The term 'C:\h\w\A5C508E5\w\A7D80973\e\mssql\Download.ps1' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

Reply:
@ilyas1974 was Local dB intentionally removed from the Windows Helix agents❔ We have a bug in our installation script either way but this may be a broader problem. Fortunately, the installation our script _should_ do places the files within the work item folder.

Reply:
Ok that makes sense, we were working because it was already installed and the download path never worked